benchmarks/bench.sh

```diff
@@ -212,6 +213,7 @@ main() {
                     run_clickbench_partitioned
                     run_clickbench_extended
                     run_imdb
+                    run_external_aggr
```

Comment:
it would be nice to add `run_external_aggr` as an option in the `data` case above too (and have it run the tpch data generator)

that way people can do
```shell
./bench.sh data external_aggr
./bench.sh run external_aggr
```